app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,124 @@
  *= require_tree .
  *= require_self
  */
+html {
+  background: rgb(0, 22, 47);
+  background: linear-gradient(0deg, rgba(0, 22, 47, 1) 0%, rgba(55, 120, 194, 1) 100%);
+  width: 100.2vw;
+  height: 100vh;
+  overflow-x: hidden;
+}
+
+#landing {
+  width: 100%;
+  height: 100%;
```

Comment:
> * As mentioned in the requirements you are to follow the given design template which includes
> 
> ```
>  Colors
> Typography: font face, size and weight.
> Layout: composition and space between elements.
> ```
> 
> * Kindly use the given UI to build your app as currently, your app doesn't follow the given design
>   ![image](https://user-images.githubusercontent.com/52497246/277073458-896cd8ac-c438-4288-859d-a88fdc8a48f7.png)
>   ![image](https://user-images.githubusercontent.com/52497246/277073479-5779990e-115d-468d-b819-e8deeac9b65d.png)
>   ![image](https://user-images.githubusercontent.com/52497246/277073512-2ddf0f8a-83c4-4832-a088-77beb8ceb48b.png)

Kindly follow the design as requested by the previous reviewer. This is a requirement for this project, and that's the reason you were given the design so that it guilds you.